clients/fides-js/src/lib/i18n/i18n-utils.ts

```diff
@@ -66,11 +68,22 @@ function extractMessagesFromExperienceConfig(
       // For each translation, extract each of the translated fields
       (translation: ExperienceConfigTranslation) => {
         const locale = translation.language;
+        const localeHasOverride =
+          experienceTranslationOverrides?.override_language === locale;
         const messages: Messages = {};
         EXPERIENCE_TRANSLATION_FIELDS.forEach((key) => {
+          const overrideValue =
+            localeHasOverride &&
+            Object.prototype.hasOwnProperty.call(
+              experienceTranslationOverrides,
+              key
+            )
+              ? // @ts-ignore
```

Comment:
This feels dangerous here. Is there a less invasive way to coerce this type to be correct? Is something misconfigured upstream forcing this need (eg convert EXPERIENCE_TRANSLATION_FIELDS to an enum)? Maybe just a comment in addition explaining why?

Reply:
👍 fixed!